File: android-exercises/10 - options_menu/OptionsMenu_java/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_java/MainActivity3.java
package com.bugrahankaramollaoglu.optionsmenu_java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity3 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main3);
    }

    public void goBack3(View view) {
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
    }
}

File: android-exercises/10 - options_menu/OptionsMenu_kotlin/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_kotlin/MainActivity3.kt
package com.bugrahankaramollaoglu.optionsmenu_kotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity3 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main3)
    }

    fun goBack3(view : View) {
        val intent : Intent = Intent(this@MainActivity3, MainActivity::class.java)
        startActivity(intent)
    }
}